contentful.config.js

```diff
@@ -3,8 +3,8 @@ const url = process.env.NEXT_PUBLIC_BASE_URL;
 module.exports = {
   contentful: {
     space_id: process.env.CONTENTFUL_SPACE_ID || '',
-    cda_token: process.env.CONTENTFUL_ACCESS_TOKEN || '',
-    cpa_token: process.env.CONTENTFUL_PREVIEW_ACCESS_TOKEN || '',
+    cda_token: process.env.CONTENTFUL_ACCESS_TOKEN_LOCAL || '',
```

Comment:
why should this become local?

Reply:
@Polleke007 shouldnt this also be dynamic? on production env you want CONTENTFUL_ACCESS_TOKEN, on lower env you want CONTENTFUL_ACCESS_TOKEN_LOCAL, no?

Reply:
This is not used, and removed